### PR TITLE
fix: handle vanilla esu form submission

### LINKED ISSUE #174: Handle 'enter' key ESU submit

### PR DESCRIPTION
**Pull request by AndyEPhipps**
Fixes #174

### CODE REVIEW

**Comment by AndyEPhipps** [comment]
Currently, hitting enter just submits the form old-school style, spitting the param into the URL.. figured i'd just put this 'fix' in for now

File: src/components/Molecules/EmailSignUp/EmailSignUp.js [hunk 1]
```diff
@@ -24,8 +24,16 @@ const EmailSignUp = ({
   ...rest
 }) => {
   const [value, setValue] = useState('');
+
+  const handleSubmit = e => {
+    // Just stopping weird things happening for now when the user submits with the enter key
+    // rather than the Submit button
+    e.preventDefault();
+    console.log('');
```
Comment (LouiaR) [inline src/components/Molecules/EmailSignUp/EmailSignUp.js]: May return something here instead of the console log

Comment (AndyEPhipps) [reply src/components/Molecules/EmailSignUp/EmailSignUp.js]: Oh yeah whoops, was editing that and didn't finish haha

**Pull request merged**